### DISCUSSION TITLE
how to change currency name?

### DISCUSSION THREAD

**Discussion by miladfa7** [dm4t2/vue-currency-input#300] [Q&A]
How can I convert currency to Rials(ریال)?

```
{
          locale: 'en-US',
          currency: 'IRR',               
  }
```
to 
```
{
          locale: 'en-US',
          currency: 'ریال',               
  }
```
It gets an error when I change currency name to ریال

help me, thanks

**Comment by dm4t2** [comment]
Try `{ currrency: 'IRR', locale: 'fa-IR' }` on the playground: https://dm4t2.github.io/vue-currency-input/playground.html

**Comment by miladfa7** [comment]
Thanks, How can I get the rial(ریال) back to the number?
default:  2,000 ریال
I want it that way 
ریال a 2,000
